lib/config/options/index.ts

```diff
@@ -567,6 +567,38 @@ const options: RenovateOptions[] = [
     default: true,
     globalOnly: true,
   },
+  {
+    name: 'inheritConfig',
+    description:
+      'If true, Renovate will inherit repo config from <orgName>/renovate-config/org-inherited-config.json5.',
+    type: 'boolean',
+    default: false,
+    globalOnly: true,
+  },
+  {
+    name: 'inheritConfigRepoName',
+    description:
+      'Renovate will look in this repo for the inheritConfigFileName.',
+    type: 'string',
+    default: '{{parentOrg}}/renovate-config',
+    globalOnly: true,
+  },
+  {
+    name: 'inheritConfigFileName',
+    description:
+      'Renovate will look for this config file name in the inheritConfigRepoName.',
+    type: 'string',
+    default: 'org-inherited-config.json5',
```

Comment:
really json5? 😕

Reply:
I was thinking it should be the default so hosted users can add comments

Reply:
this will make future config migrations harder, because we don't have a proper way to apply required changes 🤔

Reply:
yes, so your last slack comment was to always use JSON5 parser. I preferred thad some time ago, but was reverted. :wink:

Reply:
https://github.com/renovatebot/renovate/issues/27969